tests/test_coastal_vulnerability.py

```diff
@@ -1358,6 +1362,48 @@ def test_aoi_invalid_geometry(self):
                 aoi_path, landmass_path, model_resolution, target_vector_path,
                 polygon_pickle, lines_pickle, lines_rtree)
 
+    def test_prepare_landmass_invalid_geometry(self):

```

Comment:
This test I just moved from elsewhere in the file so it's close to other related tests.